src/components/UseOfFunds2024/styles/UseOfFunds.scss

```diff
@@ -25,7 +46,15 @@
 			}
 		}
 
-		&-summary-text {
+		h2.use-of-funds-heading {
+			font-weight: normal;
+			margin-bottom: 16px;
+		}
+
+		p.use-of-funds-summary-text {
+			line-height: 1.8;
+			font-size: 18px;
```

Comment:
I think this should be 16px for mobile, it looks very large on small screens and different than the rest of the UoF text

Reply:
Oh it was a typo